### ISSUE TITLE
Rejson russian letter

### ISSUE THREAD

**Issue by Pavel71** [RedisLabsModules/rejson#98]
from rejson import Client, Path 

rj = Client(host='localhost', port=6379, db=1, decode_responses=True)

with open('param_dict.json', 'rb') as file:

obj_test = json.load(file)

print(obj_test)
# Print
# {'ID': 1, 'Вес(кг.)': 17, 'Рост(см.)': 100, .....

rj.jsonset('user1', Path.rootPath(), obj_test)
print(rj.jsonget('user1'))

# Print
# {'ID': 1, 'Ð\x92ÐµÑ\x81(ÐºÐ³.)': 17, 'Ð\xa0Ð¾Ñ\x81Ñ\x82(Ñ\x81Ð¼.)':

When i set dict with reason module in redis base, russian letter looking this Ð\x92ÐµÑ\x81(ÐºÐ³.)

Is there a problem?

**Comment by mnunberg** [comment]
Try to use the `NOESCAPE` option. I'm unsure how it's done in the client.

**Comment by Pavel71** [comment]
if i use no escape option

**Comment by phdog** [comment]
Same here! I see Ð¾ÑÑÐ¸, ÐºÐ¾ÑÐ¾ÑÑÐµ Ð¿ÑÐ¾Ð²Ð sings in Keyload and when getting json back, after I SET JSON. While strings' datatype SET works well with cyrillic. I provide valid UTF8 cyrrilic input which I console.log before rejson

**Comment by mnunberg** [comment]
The issue seems to be in the Python client. As a workaround, you can try to do something like:

```
rj.execute_command('json.get', 'user1', 'noescape', '.')
```

The resultant object is JSON, which you can then pass to `json.loads`

**Comment by phdog** [comment]
No, this is definitely not a client issue. I'm using node-redis client and its ability to pass direct CLI commands via send_command(). I check JSON on its input and it's OK, and I see that I get broken string values both getting it back and in KeyLord GUI client. BTW Redis native strings work well with cyrillic. Needs a fix.

**Comment by mnunberg** [comment]
There is a known bug, which is fixed using the NOESCAPE option. We're considering making it the default even though, in theory, it may break existing behavior.

In any event, NOESCAPE (before the path) should do what you need.

**Comment by phdog** [comment]
Yes, It works in JSON.GET! Through it seems a little weird seeing kind of Ð¾ÑÑÐ chars in GUI database manager. Thank you for this module, anyway, it breathes life into Redis for me, since it makes it workable as a primary db in standard projects
